### PR TITLE
Remove MSVC project

### PR DESCRIPTION
**Pull request by Ghabry**
Guess nobody will miss these XML files.  ✨🎈🎉🎆

All Jenkins jobs already use CMake, everything should work as before even without this merged.

### CODE REVIEW

**Comment by Ghabry** [comment]
Please don't merge this yet, decided that I will also attach the CMake issues I found here. (also applies to Player)

**Comment by carstene1ns** [comment]
Sure,may also have some further ideas.

**Comment by Ghabry** [comment]
In another issue I wrote about Ehsc:

https://github.com/EasyRPG/Tools/pull/30#issuecomment-387412706

But this isn't the issue this time...

**Comment by Ghabry** [comment]
Jenkins: test this please

**Comment by Ghabry** [comment]
the Ehsc issue was resolved by removing that bigobj switch, because I forgot to use APPEND that property overwrite all compiler flags..........

File: .gitignore [hunk 1]
```diff
@@ -37,6 +30,25 @@ CMakeDoxyfile.in
...
+*.psess
+*.suo
+*.user
+*.vsp*
+*.lib
+*.exe
+*.exp
+*.ilk
+*.pdb
+install_manifest.txt
```
Comment (carstene1ns) [inline .gitignore]: this should be removed, as is generic cmake.

File: README.md [hunk 2]
```diff
@@ -60,8 +60,29 @@ To generate the "configure" script, run before following the above section:
...
+    cd liblcf-0.5.4                       # enter in the package directory
+    cmake . -DCMAKE_BUILD_TYPE=Release    # configure project
+    cmake --build .                       # compile the library
+    sudo cmake --build . --target install # install system-wide
+
+Read more detailed instructions at:
+
 * https://wiki.easyrpg.org/development/compiling/liblcf/cmake
 
+CMake is the only supported way to build liblcf for Windows.
```
Comment (carstene1ns) [inline README.md]: Should mention vcpkg here.

**Pull request merged**